README.md

```diff
@@ -88,7 +88,7 @@ traffic itself is still encrypted.
 `witchcraft-server` is configured with service, event, metric, request and trace loggers from the 
 `witchcraft-go-logging` project and emits structured JSON logs using [`zap`](https://github.com/uber-go/zap) as the
 logger implementation. The default behavior emits logs to the `var/log` directory (`var/log/service.log`, 
-`var/log/request.log`, etc.) unless the server is run in a Docker container, in which case the logs are always emitted 
+`var/log/request.log`, etc.) unless the server is run in a Docker container or a container that has an environment variable called `$CONTAINER` set, in which case the logs are always emitted
```

Comment:
nit: This would still be true if the `CONTAINER` env var is set on a non-container. So how about rewording slightly?
```
... run in a Docker container or has the environment variable `$CONTAINER` set, ...
```